### DISCUSSION TITLE
Error when rebuilding webpack.

### DISCUSSION THREAD

**Discussion by Tiensinh98** [webpack/webpack#16848] [Q&A]
Hi admin,
- I want to import a web assembly, namely "sql-wasm.wasm" for running sql.js to load an .sqlite file loaded from local machine in the browser. The sql.js needs  "sql-wasm.wasm" file for instantiation since it cannot immediately instantiate the database by loading a .sqlite file.
- Since that, I need to add `{ test: /\.wasm$/, type: "webassembly/async"} into the rules section inside modules and `experiments: {asyncWebAssembly: true}` in the experiments section of the webpack.config.json.
-  I'm working with the power bi custom visual that does not allow to modify the webpack.config.json directly, but I have to use a tool to rebuild webpack.
- Since, I didn't see the webpack.config.json the the pbiviz project, i searched on the Microsoft Power BI Community and found a related question (https://community.powerbi.com/t5/Developer/custom-visual-where-is-the-webpack-config-file/m-p/1391355). I figured out that I have to install powerbi-visuals-webpack-plugin to build my own webpack.
- I follow completely the instructions for installing powerbi-visuals-webpack-plugin (https://www.npmjs.com/package/powerbi-visuals-webpack-plugin?activeTab=readme). However, when I run command `npm run package` to create a visual package, it gave me the errors below: 
![image](https://user-images.githubusercontent.com/91424037/226558570-c71e0577-9ae8-4982-ae33-97d4bff61b54.png)
Please help!!!

**Comment by alexander-akait** [comment]
Can you create small reproducible test repo, very strange
